src/app/video-alumno/video-alumno.component.ts

```diff
@@ -56,32 +58,26 @@ export class VideoAlumnoComponent {
   }
 
   async savePlayer(player) {
+    this.videoStateHandler.reset();
     this.player = player;
-    console.log("Player instance", player);
-    this.getContentMark();
```

Comment:
Este llamado a `this.getContentMark();` se está perdiendo

Reply:
esta en la linea 63